core/src/main/java/com/scalar/db/storage/rpc/GrpcTableMetadataManager.java

```diff
@@ -105,4 +105,16 @@ public int hashCode() {
       return Objects.hash(namespace, table);
     }
   }
+
+  @Override
+  public void deleteTableMetadata(String namespace, String table) {
+    // TODO To implement
```

Comment:
Since we don't expose `deleteTableMetadata` and `addTableMetadata` gRPC endpoints, `GrpcTableMetadataManager` will support read only operations. So we don't need this `TODO`.
```suggestion
```